main.js

```diff
@@ -27,6 +27,7 @@ class Accuweather extends utils.Adapter {
 		super({
 			...options,
 			name: "accuweather",
+			strictObjectChecks: false 
```

Comment:
@algar42 That should NOT be the solution! YOu please should adjust the code so that objects are created before you set States. In worst case - as the message said - the values will not be written in a future version

Reply:
All the required objects are created in advance. The log warning appears for some API xml fields that are not expected to be created as object, so this solution is ok in this case.. But when i have some time I'll take a look at the code to find a way not to write to nonexisting object